react/features/overlay/components/web/AbstractPageReloadOverlay.js

```diff
@@ -118,7 +100,7 @@ export default class AbstractPageReloadOverlay<P: Props>
      * @returns {boolean} - If this overlay needs to be rendered, {@code true};
      * {@code false}, otherwise.
      */
-    static needsRenderWeb(state: Object) {
+    static needsRender(state: Object) {
```

Comment:
Isn't the logic here and the one in the new native dialog the same? would it make sense to have some `getFatalError(state)` helper perhaps?

Reply:
I think isErrorFatal(state) because it would return true or false based on if the error is fatal or not. The one from the native dialog checks the type of the error so it can update the reason string value.

Reply:
I said get because in case it IS fatal you want the error object to extract the type etc, right? So it would help with both knowing if there is a fatal error and then extracting the info about it.